ui/server/src/main/scala/pl/touk/nussknacker/ui/process/ProcessObjectsFinder.scala

```diff
@@ -75,4 +98,21 @@ object ProcessObjectsFinder {
     val processesOnly =  allProcesses.filter(!_.properties.isSubprocess)
     val subprocessesOnly = allProcesses.filter(_.properties.isSubprocess)
   }
+}
+
+case class ProcessComponent (processName: String, nodeId: String, nodeType: String, processCategory: String, isDeployed: Boolean)
+
+object ProcessComponent {
```

Comment:
we have WithComponent trait, let's use that? and use componentId consistently instead of type?